modelarmor/api/ModelArmor.Samples.Tests/ModelArmor.Samples.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Google.Cloud.Dlp.V2" Version="4.18.0" />
```

Comment:
You don't need this dependecy here, you get it transitively from the samples project.